src/component/scramblename.jsx

```diff
@@ -0,0 +1,17 @@
+import { useState } from "react";
+import { useEffect } from "react";
+
+export const ScramblePokeName = ({ currPokemon, setCurrentPokemon }) => {
+  const [scrambledName, setScrambledName] = useState("");
+
+  useEffect(() => {
+    const scramble = () => {
+      const letters = currPokemon.split("");
+      const shuffledLetters = letters.sort(() => 0.5 - Math.random());
```

Comment:
this is where I would want to see a comment, as the math here does not mean anything to me reading the code